Sources/PointFree/Database.swift

```diff
@@ -91,17 +92,15 @@ private func fetchUser(with token: GitHub.AccessToken) -> EitherIO<Error, Databa
     [token.accessToken]
     )
     .map { result -> Database.User? in
-      let uuid = result[3, "id"].flatMap(get(\.string) >>> flatMap(UUID.init(uuidString:)))
-
-      let subscriptionId = result[5, "subscription_id"]
-        .flatMap(get(\.string) >>> flatMap(UUID.init(uuidString:)))
+      let uuid = result["id"]?.array?.first?.wrapped.string.flatMap(UUID.init(uuidString:))
```

Comment:
lol I really hope there is a better way but this seems to be the way to get this data from the `result`?!

Reply:
😨